ros2bag/ros2bag/verb/record.py

```diff
@@ -71,7 +89,13 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
         )
         parser.add_argument(
             '--include-hidden-topics', action='store_true',
-            help='record also hidden topics.')
+            help='record also hidden topics.'
+        )
+        parser.add_argument(
+            '--qos-profile-overrides', type=str, default='',
```

Comment:
The correct type here is `argparse.FileType('r')`.
See https://docs.python.org/3/library/argparse.html#argparse.FileType